students/kosarzewski_maciej/lesson_01_basics/total_cost.py

```diff
@@ -0,0 +1,7 @@
+# Read an integer:
+A = int(input())
```

Comment:
Capital letters should be used only for constants e.g.
MY_NAME = "Kamil"

Please use lower case with underscores for variable names instead: